hazelcast-spring/src/main/resources/hazelcast-spring-5.0.xsd

```diff
@@ -1003,6 +1005,8 @@
                                         </xs:documentation>
                                     </xs:annotation>
                                 </xs:attribute>
+                                <!-- we already have 'hot-restart' (and now, data-persistence),
+                                    why do we need this? It does not even exist in hazelcast-config-5.0 -->
```

Comment:
I would not commit with such a comment. Can you please elaborate on this and remove it if it is not needed?

Reply:
I looked a bit more into it, and it seems we can remove it. Basically, cache configuration in xsd has this attribute `hot-restart-enabled` which is neither parsed by `HazelcastConfigBeanDefinitionParser` nor used anywhere else. The reason is it has been moved into `hot-restart`. Looked into git history logs for `CacheSimpleConfig` (where this attribute would have been used had it been parsed) and back in 2015 it had field `private boolean hotRestartEnabled;` only to be removed and replaced by `HotRestartConfig` shortly after its introduction.